### ISSUE TITLE
Linting for pipes in `pdf()` or `postscript()`

### ISSUE THREAD

**Issue by TimTaylor** [jimhester/lintr#3059]
E.g. 

```R
postscript("|lp -dlw")
pdf("|lp -o landscape", paper = "a4r")
```

Motivated by the following report https://bugs.r-project.org/show_bug.cgi?id=19087 and https://github.com/r-devel/r-svn/pull/264. This seems like something lintr could flag.

**Comment by MichaelChirico** [comment]
I would wait for R-core to weigh in here.

In the meantime, it looks like flowr already has something set up for this:

https://github.com/flowr-analysis/flowr/blob/983644de009dd3f0abfe19f34a825fb304ec89df/test/functionality/linter/lint-problematic-inputs.test.ts#L44

**Comment by TimTaylor** [comment]
Actually it’s probably worth closing this (I can’t now as away from pc). It’s not really something to lint for as it’s really about sanitising untrusted inputs. I don’t think you will see any hits in packages as more of an interactive convenience type thing. Personally think it’s more a case of improving the pdf documentation over anything else (it’s well buried there but clearer in postscript).
